modules/customize-certificates-api-update-named.adoc

```diff
@@ -0,0 +1,92 @@
+// Module included in the following assemblies:
+//
+// * security/certificates/api-server.adoc
+
+:_content-type: PROCEDURE
+[id="customize-certificates-api-update-named_{context}"]
+= Update an API server named certificate
+
+Updating the API server certificate to new version requires only a creation of a new secret and updating the `apiserver`.
+
+.Procedure
+
+. Login to the API as the `kubeadmin` user.
++
+[source,terminal]
+----
+$ oc login -u kubeadmin -p <password> https://FQDN:6443
+----
+
+. Create a secret with new name that contains the certificate chain and private key in the `openshift-config` namespace.
++
+[source,terminal]
+----
+$ oc create secret tls <secret> \//<1>
+     --cert=</path/to/cert.crt> \//<2>
+     --key=</path/to/cert.key> \//<3>
+     -n openshift-config
+----
+<1> `<secret>` is the name of the secret that will contain the certificate chain and private key.
+<2> `</path/to/cert.crt>` is the path to the certificate chain on your local file system.
+<3> `</path/to/cert.key>` is the path to the private key associated with this certificate.
++
+[NOTE]
+====
+Do not overwrite or delete the previous secret. Create a secret with a new name, for example `api-certs-<date>`. In case of the rollback, you can just switch back to the original secret name in the `apiserver/cluster`.
+====
+
+. Update the API server to reference the created secret with new API certificate.
++
+[source,terminal]
+----
+$ oc patch apiserver cluster \
+     --type=merge -p \
+     '{"spec":{"servingCerts": {"namedCertificates":
+     [{"names": ["<FQDN>"], //<1>
+     "servingCertificate": {"name": "<secret>"}}]}}}' <2>
+----
+<1> Replace `<FQDN>` with the FQDN that the API server should provide the certificate for.
+<2> Replace `<secret>` with the name used for the secret in the previous step.
+
+. Examine the `apiserver/cluster` object and confirm the secret is now
+referenced.
++
+[source,terminal]
+----
+$ oc get apiserver cluster -o yaml
+----
++
+.Example output
+[source,terminal]
+----
+...
+spec:
+  servingCerts:
+    namedCertificates:
+    - names:
+      - <FQDN>
+      servingCertificate:
+        name: <secret>
+...
+----
+
+. Verify that the new certificate is in place.
++
+[source,terminal]
+----
+$ curl -kv https://api.<cluster_name>.<base_domain> 2>&1 | grep -A4 -i "server certificate"
```

Comment:
Can you add some example output of what the user should expect to see, to know that it is in place?

Reply:
Added, please PTAL

Reply:
LGTM thanks!